cookbooks/adiabatic_boundary.prm

```diff
@@ -0,0 +1,80 @@
+set Dimension                              = 3
+
+set Use years in output instead of seconds = true
+set End time                               = 0
+set Nonlinear solver scheme                = single Advection, single Stokes
+set Nonlinear solver tolerance             = 1e-5
+set Linear solver tolerance                = 1e-5
```

Comment:
This will need to be moved to the new (since ASPECT 2.0) Solver parameters subsection.

The subsection will have the following format:
```
subsection Solver parameters
  subsection Stokes solver parameters
    set Linear solver tolerance = 1e-5
  end
end
```

Reply:
Have you tested the model with a stricter Linear solver tolerance? For example, do the results change significantly if you use the default value of 1e-7?